volatility3/framework/plugins/linux/kmsg.py

```diff
@@ -136,7 +136,12 @@ def symtab_checks(cls, vmlinux: interfaces.context.ModuleInterface) -> bool:
         """
 
     def get_string(self, addr: int, length: int) -> str:
-        txt = self._context.layers[self.layer_name].read(addr, length)  # type: ignore
+        layer = self._context.layers[self.layer_name]
+        if not layer.is_valid(addr, length):
+            return ""
```

Comment:
Would an exception or `None` not be better than returning a blank string?  My concern is the string having a length of 0 or similar, and no indication that it failed otherwise...